### PR TITLE
update definition of sb based on reviewer comments

### PR DESCRIPTION
**Pull request by chantelwetzel-noaa**
Revised the definition of SB based on reviewer comments.

### CODE REVIEW

File: R/SpawningBiomass.R [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-#' of reproductive age during the breeding season of a
-#' stock. Most often used as a proxy for measuring
-#' egg production, the SSB depends on the abundance of
-#' the various age classes composing the stock and their
-#' past exploitation pattern, rate of growth, fishing and
-#' natural mortality rates, onset of sexual maturity, and
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:5]: weight-at-age should be "weight at age"

File: R/SpawningBiomass.R [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-#' stock. Most often used as a proxy for measuring
-#' egg production, the SSB depends on the abundance of
-#' the various age classes composing the stock and their
-#' past exploitation pattern, rate of growth, fishing and
-#' natural mortality rates, onset of sexual maturity, and
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion 
+#' mature-at-age. Alternatively, it can be defined as the biomass of
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:6]: mature-at-age should be mature at age

File: R/SpawningBiomass.R [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-#' the various age classes composing the stock and their
-#' past exploitation pattern, rate of growth, fishing and
-#' natural mortality rates, onset of sexual maturity, and
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion 
+#' mature-at-age. Alternatively, it can be defined as the biomass of 
+#' all individuals at or above “age at 50 percent maturity” or “size 
+#' at 50 percent maturity.” or the total biomass of fish of reproductive
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:8]: remove the full stop after maturity

File: R/SpawningBiomass.R [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
-#' past exploitation pattern, rate of growth, fishing and
-#' natural mortality rates, onset of sexual maturity, and
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion 
+#' mature-at-age. Alternatively, it can be defined as the biomass of 
+#' all individuals at or above “age at 50 percent maturity” or “size 
+#' at 50 percent maturity.” or the total biomass of fish of reproductive 
+#' age during the breeding season of a stock. Most often used as a proxy
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:9]: Remove "of a stock"

File: R/SpawningBiomass.R [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion 
+#' mature-at-age. Alternatively, it can be defined as the biomass of 
+#' all individuals at or above “age at 50 percent maturity” or “size 
+#' at 50 percent maturity.” or the total biomass of fish of reproductive 
+#' age during the breeding season of a stock. Most often used as a proxy 
+#' for measuring egg production, the spawning biomass depends on the 
+#' abundance of the various age classes composing the stock and their
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:11]: change stock to population

File: R/SpawningBiomass.R [hunk 6]
```diff
@@ -1,18 +1,16 @@
...
-#' natural mortality rates, onset of sexual maturity, and
-#' environmental conditions.
+#' The mass of fish (males and females or females only) in the 
+#' population that contribute to reproduction. Often conventionally 
+#' defined as the product of weight-at-age and the proportion 
+#' mature-at-age. Alternatively, it can be defined as the biomass of 
+#' all individuals at or above “age at 50 percent maturity” or “size 
+#' at 50 percent maturity.” or the total biomass of fish of reproductive 
+#' age during the breeding season of a stock. Most often used as a proxy 
+#' for measuring egg production, the spawning biomass depends on the
```
Comment (kellijohnson-NOAA) [inline R/SpawningBiomass.R:10]: "Most often used as a proxy for measuring egg production" does not seem to be tied to the rest of the sentence 🤷 . With the comma after the clause it appears to be a qualifying statement but I am not seeing the connection. Perhaps make it its own sentence and put it at the end of the paragraph.

File: man/Catch.Rd
Comment (kellijohnson-NOAA) [inline man/Catch.Rd:1]: I am guessing that this change in Catch.Rd is a byproduct of some other commit without the documentation being rendered? Perhaps we should move it into its own commit?

**Review by kellijohnson-NOAA — CHANGES_REQUESTED** [review]
There are a few typos but the biggest thing that I found was the use of "stock" where we want to use population.

File: R/SpawningBiomass.R (hunk 4 above)
Comment (chantelwetzel-noaa) [reply R/SpawningBiomass.R:9]: That is our bad, since it was in the original text too.  I agree that this should be revised.

**Comment by chantelwetzel-noaa** [comment]
@kellijohnson-NOAA I have separated out the catch edits to its own commit and have made the suggested edits for spawning biomass.

**Review by kellijohnson-NOAA — APPROVED** [review]

**Comment by chantelwetzel-noaa** [comment]
@Bai-Li-NOAA I think me question to you in this review is about the Shiny page.  Should the process to be to publish the revised webpage post-merging in the pull request or does that happen automatically with the github actions?  Thank you

**Comment by Bai-Li-NOAA** [comment]
@chantelwetzel-noaa, this is embarrassing, I don't remember how it works. @ChristineStawitz-NOAA might be able to provide a better answer to this question.

**Comment by k-doering-NOAA** [comment]
@chantelwetzel-noaa and @Bai-Li-NOAA , sorry for lurking, but I think I can help a bit with this one!

I just gave both of you edit permissions on the Shiny app on Posit connect. 

It looks like it was published to deploy from a branch called shiny_app and to check automatically for updates on the branch. However, I think the branch no longer exists, so it can't pull in updates. 
image


Therefore, it may need to be re-deployed from an existing branch of your choosing, if you would like it to stay automatically up to date. One note is that the manifest file may need to be updated periodically (either manually and pushed to the branch, or through a github action). I didn't look at the ghas in this repo, so apologize if this is already being dealt with!

**Comment by chantelwetzel-noaa** [comment]
@k-doering-NOAA You are the kind of lurker that everyone loves.  Thank you for taking the time to look at this.  I'll see If I can get the set back up based on your suggestions.

**Pull request merged**